### DISCUSSION TITLE
Failure Testing

### DISCUSSION THREAD

**Discussion by ivan-curkovic** [docker/extension-ideas#131] [Ideas]
This ideas was submitted via Docker Desktop Extension Feedback form: 

Net fault testing as described in https://www.qualimente.com/2016/04/26/introduction-to-failure-testing-with-docker/ would enable me to perform network resiliency testing more conveniently.

**Comment by LyndonGingerich** [comment]
Thanks, @ivan-curkovic!

I submitted the request and will gladly provide further details to the best of my ability.
